function/Dockerfile.armhf

```diff
@@ -0,0 +1,57 @@
+ARG RASPBIAN_VERSION=stretch
+FROM resin/rpi-raspbian:$RASPBIAN_VERSION
+
+# update apt
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends apt-utils \
+	# install necessary build tools \
+	&& apt-get -qy install build-essential cmake pkg-config unzip wget make \
+	# install necessary libraries \
+	&& apt-get -qy install \
+		libgtk2.0-dev \
+		libgtk-3-dev \
+		libatlas-base-dev \
+		gfortran \
+		python2.7-dev \
+		python3-dev \
+		python-pip \
+		python-numpy \
+		python3-pip \
+		python3-numpy \
+        python3-markdown \
+        python3-networkx \
+        python3-h5py \
+        python3-lxml \
+        python3-matplotlib \
+        python3-protobuf \
+        python3-dateutil \
+        python3-scipy \
+        python3-six \
+		libraspberrypi0 \
+		python-setuptools \
+		python3-setuptools \
+	# cleanup apt. \
+	&& apt-get purge -y --auto-remove \
+	&& rm -rf /var/lib/apt/lists/*
+
+WORKDIR /root/
+
+COPY requirements.txt ./
+
+RUN pip3 install -r requirements.txt
+
+RUN wget https://storage.googleapis.com/download.tensorflow.org/deps/pi/2018_03_22/tensorflow-1.6.0-cp35-none-any.whl 
```

Comment:
can we && these instructions for fewer layers with `rm tensorflow-1.6.0-cp35-none-any.whl ` being part of that same block?

Reply:
Since the pip req install is so tightly coupled to tensorflow, probably better to also group the installation of the wheel file itself too (the layer right after L43 as well..).